test/integration/api/services_controller_test.rb

```diff
@@ -25,15 +28,15 @@ class Api::ServicesControllerTest < ActionDispatch::IntegrationTest
   end
 
   test 'update the settings' do
-    put admin_service_path(service), update_params
+    put admin_service_path(service), params: update_params
     assert_equal 'Service information updated.', flash[:notice]
 
     update_service_params = update_params[:service]
     expected_notification_settings = update_service_params[:notification_settings].transform_values { |notifications| notifications.map(&:to_i) }
     expected_buyer_plan_change_permission = update_service_params[:buyer_plan_change_permission]
     expected_signup_and_use = update_service_params
-                                  .slice(:intentions_required, :buyers_manage_apps, :referrer_filters_required, :custom_keys_enabled, :buyer_can_select_plan)
```

Comment:
This is probably your IDE's fault, I think it is more readable with the indentation.